examples/check_combobox.py

```diff
@@ -0,0 +1,41 @@
+from qtpy.QtWidgets import QApplication, QMainWindow, QPushButton, QVBoxLayout, QWidget
+
+from src.superqt import QCheckComboBox
```

Comment:
```suggestion
from superqt import QCheckComboBox
```

(if this wasn't working for you, then it's likely you hadn't run `pip install -e .`)

Reply:
Thanks for catching that. Actually, I usually develop while the example is in the root directory without using the `pip install -e .`. Reading more about it, it looks like a better substitute than my method :) I will fix that.

Reply:
Fixed in latest commit